docs/it-manual/sre-playbook/terraform-deploy-system.md

```diff
@@ -16,7 +18,11 @@ You will need couple of values that are configured outside of Civiform before yo
 3. Find the version that you want to deploy on [Github](https://github.com/civiform/civiform/releases)
 4. Copy the civiform\_config.example.sh into civiform\_config.sh and fill out the missing values. You can get a sense of required values depending on your cloud provider by looking at [staging-azure](https://github.com/civiform/staging-azure-deploy/blob/main/civiform_config.sh) or [staging-aws](https://github.com/civiform/staging-aws-deploy/blob/main/civiform_config.sh) configs.
 5. Run the `bin/doctor --tag=<version>` and install the dependencies
-6. Run `bin/setup --tag=<version>`
+6. Run `bin/setup --tag=<version>`. What to expect:
+    * Runs 5-10 minutes.
+    * Terraform brings up resources in cloud (database, network, server, etc).
+    * Asks confirmation few times before creating resources listing everything that will be created. 
+    * Safe to re-run script if it fails.
```

Comment:
Maybe worth adding that until https://github.com/cn-terraform/terraform-aws-logs-s3-bucket/issues/6 is resolved, setup will have to be run twice

Reply:
Done.